.github/workflows/newfold-prep-release.yml

```diff
@@ -4,37 +4,29 @@ on:
   workflow_dispatch:
     inputs:
       level:
-        description: The level of release to be used.
+        description: "The level of release to be used."
```

Comment:
Can we use single quotes for strings in YAML files please, @circlecube? Double quotes can sometimes apply some escaping unexpectedly.

Reply:
Yes, updated.